Changelogs for Digital Twin Manager UI 1025.0.1 (#4548)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-0-1-improve-invalid-sharing-mode-warning.md

---
date: ""

> title: "Improve invalid sharing mode warning."

Reviewer: ```suggestion
title: "Improved invalid sharing mode warning"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2845"
version: "1025.0.1"
---

> Improves the warning shown when the sharing mode is invalid by

Reviewer: ```suggestion
The warning shown when the sharing mode is invalid has been improved. Instead of displaying a long inline message
```

> displaying a short label with an info icon instead of a long inline

Reviewer: ```suggestion
a short label with an info icon is now displayed.
```

> message.

Reviewer: ```suggestion
```
